connectors/api/datacatalog.spec.yaml

```diff
@@ -94,4 +94,37 @@ paths:
           '404':
             description: id not found
           '401':
-            description: Unauthorized
\ No newline at end of file
+            description: Unauthorized
+
+  /updateAsset:
+      put:
+        summary: This REST API updates data asset information to the data catalog configured in fybrik
```

Comment:
I'm not a native speaker but looks like this sentence is wrong. 
This REST API updates data asset information `in` the data catalog

The same for the Readmy files.

Reply:
fixed. Thanks.